common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -33,12 +34,12 @@ def visit_account_settings_page(self):
         self.account_settings_page.visit()
         self.account_settings_page.wait_for_ajax()
 
-    def log_in_as_unique_user(self, email=None):
+    def log_in_as_unique_user(self, email=None, full_name=None):
         """
         Create a unique user and return the account's username and id.
         """
         username = "test_{uuid}".format(uuid=self.unique_id[0:6])
-        auto_auth_page = AutoAuthPage(self.browser, username=username, email=email).visit()
+        auto_auth_page = AutoAuthPage(self.browser, username=username, email=email, full_name=full_name).visit()
```

Comment:
This change was necessary because the test wants to verify how the user's full name displays/changes. There is currently no way to get back the full name from AutoAuthPage, so I either had to add that ability, or allow the passing in of an explicit full name. It seemed more useful to allow tests to specify an explicit full name.